ROVU/Core/src/robot/A_Star.java

```diff
@@ -93,23 +80,26 @@ private static Node findRoute(){
 
 	}
 	
-	
-	private static void neighborFringe (Node node){
+	/**
+	 * 
+	 * @param node
+	 */
+	private  void neighborFringe (Node node){
 		int i=0;
-		tk = new TreeMap<Double, Node>();
-		System.out.println("Null "+node.roomID);
+
+		System.out.println("Null "+node.nodeID);
 		for (Node e:node.neighbours){
 			if (!listContains(e,closed) && !e.wall){
-				Node temp = new Node(false, false, false, e.roomID, e.point);
+				Node temp = new Node(false, false, false, e.nodeID, e.point);
 				temp.parent=node;
 				temp.neighbours=e.neighbours;
```

Comment:
`temp.setNeighbors(e.getNeighbors)` 

Additionally, be consistent with the spelling of `neighbor`. Use either `neighbour` or `neighbor` in the entire project, not both.